filebeat/prospector/prospector_log.go

```diff
@@ -32,6 +32,10 @@ func NewLog(p *Prospector) (*Log, error) {
 		config:     p.config,
 	}
 
+	if len(p.config.Paths) == 0 {
+		return nil, fmt.Errorf("Each prospector must have at least one path defined.")
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
error strings should not be capitalized or end with punctuation or a newline